### PR TITLE
added doc for using torch.distributed.launch/run

### PR DESCRIPTION
**Pull request by oroojlooy**
Following up on https://github.com/lvwerra/trl/issues/317, I created this PR.

### CODE REVIEW

File: docs/source/customization.mdx [hunk 1]
```diff
@@ -162,4 +178,20 @@ if ds_plugin is not None and ds_plugin.is_zero3_init_enabled():
         sentiment_pipe = pipeline("sentiment-analysis", model="lvwerra/distilbert-imdb", device=device)
 else:
     sentiment_pipe = pipeline("sentiment-analysis", model="lvwerra/distilbert-imdb", device=device)
-```
\ No newline at end of file
+```
+
+## Use torch distributed 
+torch.distributed package provides PyTorch natives method to distribute a network over several machines (mostly usefule if there are several GPU nodes). It copies the model on each GPU, runs the forward and backward on each and then applies the mean of gradient of all GPUs for each one. If running torch 1.XX, you can call `torch.distributed.launch`, like
```
Comment (younesbelkada) [inline docs/source/customization.mdx:184]:
```suggestion
torch.distributed package provides PyTorch natives method to distribute a network over several machines (mostly useful if there are several GPU nodes). It copies the model on each GPU, runs the forward and backward on each and then applies the mean of gradient of all GPUs for each one. If running torch 1.XX, you can call `torch.distributed.launch`, like
```

**Review by younesbelkada — APPROVED** [review]
Thanks a lot for this!
For some reason the diff also shows that you have added the paragraph `### Run on multiple GPUs / node`, but it seems that everything is correct as expected when looking at the generated documentation. Could you maybe merge your branch with `trl`'s main branch?

cc @lvwerra

**Comment by oroojlooy** [comment]
> Thanks a lot for this! For some reason the diff also shows that you have added the paragraph `### Run on multiple GPUs / node`, but it seems that everything is correct as expected when looking at the generated documentation. Could you maybe merge your branch with `trl`'s main branch?
> 
> cc @lvwerra

That was weird for me as well. I cloned the repo this morning to have a clean commit. Do you want me to re-do that with a new clone?

**Comment by younesbelkada** [comment]
I have updated your branch for you! Will merge once it is green

**Pull request merged**